test/test_definition.jl

```diff
@@ -0,0 +1,207 @@
+module test_definition
+
+using Test
+using JETLS
+using JETLS: JL, JS, get_best_node, method_definition_range, definition_locations,
+             offset_to_xy
+
+function get_cursor_positions(code::AbstractString; cursor::AbstractString="│")
+    offsets = Int[]
+    while true
+        pos = findfirst(cursor, code)
+        if pos === nothing
+            break
+        end
+        push!(offsets, pos.start)
+        code = replace(code, cursor => "", count=1)
+    end
+    return offsets, replace(code, cursor => "")
+end
```

Comment:
Maybe best to use `JETLS.get_text_and_positions`. It handles cases where multiple `matcher`s are found in the given `text`, although your test cases don't need that.

Reply:
I’ve updated the code to use this: 825554b